sentry/project_filter.go

```diff
@@ -15,22 +14,23 @@ type ProjectFilter struct {
 
 // ProjectOwnershipService provides methods for accessing Sentry project
 // filters API endpoints.
-type ProjectFilterService struct {
-	sling *sling.Sling
-}
+type ProjectFilterService service
 
-func newProjectFilterService(sling *sling.Sling) *ProjectFilterService {
-	return &ProjectFilterService{
-		sling: sling,
+// Get the filters.
+func (s *ProjectFilterService) Get(ctx context.Context, organizationSlug string, projectSlug string) ([]*ProjectFilter, *Response, error) {
+	url := fmt.Sprintf("0/projects/%v/%v/filters/", organizationSlug, projectSlug)
+	req, err := s.client.NewRequest("GET", url, nil)
```

Comment:
It would be better if you use `http.MethodGet` instead of hardcoding "GET".
But then it might not match the rest of the codes?! Up to you.
https://pkg.go.dev/net/http#pkg-constants

Reply:
Let's nudge the author into better practices :P https://github.com/Canva/go-sentry/pull/3/commits/eb3a516a4eb089d8e993bdd6f81a937b21db7df3